doc/source/release-notes/release-notes_index.rst

```diff
@@ -1,45 +1,82 @@
-==================================
-|Percona Server| 5.7 Release notes
-==================================
+=============================================
+*Percona Server for MySQL* 5.7 Release notes
+=============================================
 
 .. toctree::
    :maxdepth: 1
 
-   Percona-Server-5.7.35-38
-   Percona-Server-5.7.34-37
-   Percona-Server-5.7.33-36
-   Percona-Server-5.7.32-35
-   Percona-Server-5.7.31-34
-   Percona-Server-5.7.30-33
-   Percona-Server-5.7.29-32
-   Percona-Server-5.7.28-31
-   Percona-Server-5.7.27-30
-   Percona-Server-5.7.26-29
-   Percona-Server-5.7.25-28
-   Percona-Server-5.7.24-27
-   Percona-Server-5.7.24-26
-   Percona-Server-5.7.23-25
-   Percona-Server-5.7.23-24
-   Percona-Server-5.7.23-23
-   Percona-Server-5.7.22-22
-   Percona-Server-5.7.21-21
-   Percona-Server-5.7.21-20
-   Percona-Server-5.7.20-19
-   Percona-Server-5.7.20-18
-   Percona-Server-5.7.19-17
-   Percona-Server-5.7.18-16
-   Percona-Server-5.7.18-15
-   Percona-Server-5.7.18-14
-   Percona-Server-5.7.17-13
-   Percona-Server-5.7.17-12
-   Percona-Server-5.7.17-11
-   Percona-Server-5.7.16-10
-   Percona-Server-5.7.15-9
-   Percona-Server-5.7.14-8
-   Percona-Server-5.7.14-7
-   Percona-Server-5.7.13-6
-   Percona-Server-5.7.12-5
-   Percona-Server-5.7.11-4
-   Percona-Server-5.7.10-3
-   Percona-Server-5.7.10-2
-   Percona-Server-5.7.10-1
+   Percona Server for MySQL 5.7.35-38 (2021-08-18) <Percona-Server-5.7.35-38.rst>
```

Comment:
Percona Server for MySQL 5.7.37-40
2022-03-31
Percona Server for MySQL 5.7.36-39
Date:	December 22, 2021

Reply:
fixed